packages/playground/src/components/node_details_cards/node_details_card.vue

```diff
@@ -53,6 +53,13 @@ export default {
           name: "Number of Deployments",
           value: props.node.stats && props.node.stats.system ? props.node.stats.users.deployments.toString() : "N/A",
         },
+        {
+          name: "Last Deployment",
+          value:
+            props.node.stats && props.node.stats.system
+              ? toHumanDate(props.node.stats.users.last_deployment_timestamp)
+              : "N/A",
+        },
```

Comment:
you checking `props.node.stats` and `props.node.stats.system` then access `props.node.stats.users` ?

Also you can use Optional chaining
```ts
props.node?.stats?.users?.last_deployment_timestamp ?? "N/A"
```